### ISSUE TITLE
Page content change is not announced when clicking “Manage/View submissions”

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1707]

**Comment by qtamu** [comment]
This should be compliant to [WCAG 2.1 Level AA Guidelines (4.1.3 – Status Messages)](https://www.w3.org/TR/WCAG22/#dfn-assistive-technologies) The screen reader I tested with is NVDA for windows os. Once the link is clicked the screen reader says a whole bunch of things on the screen then ends with "visited link" to signify that the user is visiting page of the link they just clicked on.

Not sure If this is what the desired behavior is. Please test with other screen readers in other environments. There is a way to try to configure what a screen reader says when a link is clicked but that requires more thought.
